src/Sulu/Bundle/MediaBundle/Content/Types/MediaSelectionContentType.php

```diff
@@ -44,25 +45,29 @@ public function __construct($mediaManager, $template)
     public function getDefaultParams()
     {
         return array(
-            'types' => null,
-            'defaultDisplayOption' => 'top',
-            'displayOptions' => array(
-                'leftTop' => true,
-                'top' => true,
-                'rightTop' => true,
-                'left' => true,
-                'middle' => false,
-                'right' => true,
-                'leftBottom' => true,
-                'bottom' => true,
-                'rightBottom' => true
+            'types' => new PropertyParameter('types', 'string', null),
+            'defaultDisplayOption' => new PropertyParameter('defaultDisplayOption', 'string', 'top'),
+            'displayOptions' => new PropertyParameter(
+                'displayOptions',
+                'collection',
+                array(
+                    'leftTop' => new PropertyParameter('leftTop', 'string', true),
+                    'top' => new PropertyParameter('top', 'string', true),
+                    'rightTop' => new PropertyParameter('rightTop', 'string', true),
+                    'left' => new PropertyParameter('left', 'string', true),
+                    'middle' => new PropertyParameter('middle', 'string', false),
+                    'right' => new PropertyParameter('right', 'string', true),
+                    'leftBottom' => new PropertyParameter('leftBottom', 'string', true),
+                    'bottom' => new PropertyParameter('bottom', 'string', true),
+                    'rightBottom' => new PropertyParameter('rightBottom', 'string', true)
```

Comment:
Storing true as a string feels quite strange... Shouldn't we make the type nullable, and only have collection and such types?